official/update.rkt

```diff
@@ -148,13 +148,15 @@
     (λ (get-info)
       (if get-info
         (list (pkg:extract-pkg-dependencies get-info)
+              (pkg:extract-pkg-dependencies get-info #:build-deps? #f)
               (get-info 'implies (λ () empty))
               (get-info 'collection (λ () #f)))
         (list empty empty #f)))))
```

Comment:
I think this line is now an element short.  Maybe it should be
```
(list empty empty empty #f)
```

Reply:
Whoops, you are right!